src/js/gi-bill-comparison-tool/components/Dropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+class Dropdown extends React.Component {
```

Comment:
I'm not too familiar with all of our common React components, but we do have one for selects, could that be used here, or extended to add in any missing functionality?